istio-operator/istio-operator.spec

```diff
@@ -31,9 +39,10 @@ Version:        0.10.0
 Release:        1%{?dist}
```

Comment:
Do I need to change this from `1` to `2`?

Reply:
This isn't released yet, so I wouldn't worry about it. This should be incremented with rebuilds.

Reply:
let me update the revision history then

Reply:
updated.